components/builder-sessionsrv/src/migrations/sessions.rs

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+// DO NOT USE THIS TABLE - IT HAS BEEN DEPRECATED AND DROPPED
```

Comment:
You probably don't need this comment at the top - either remove or bring it closer to the actual table it refers to..

Reply:
I was looking for a way to signify that this file should no longer be used for migrations.

Reply:
Maybe a better note would be a TBD to delete this file entirely after migration cleanup is done..

Reply:
I like that idea.